frontend/docs/pages/home/features/child-workflows.mdx

```diff
@@ -6,8 +8,24 @@ Note that spawning child workflows is a **durable** operation, meaning that spaw
 
 ## Looping Workflows
 
-To loop through a list and create a new workflow per element in the list, you can simply use `spawnWorkflow` in a loop. For example, in the Typescript SDK:
+To loop through a list and create a new workflow per element in the list, you can simply use `spawnWorkflow` in a loop. For example:
+
+<Tabs items={['Python', 'Typescript', 'Go']}>
+  <Tabs.Tab>
+```python
+@hatchet.workflow("fanout:create")
+class Parent:
+    @hatchet.step()
+    def spawn(self, context: Context):
+        for i in range(10):
+          context.spawn_workflow(
```

Comment:
@abelanger5 do we want to document the aio client here?

Reply:
The aio client is mentioned in the scatter gather workflows further down, we can expand on it there as well.